### PR TITLE
Squeakr

### PR DESCRIPTION
**Pull request by rob-p**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by rob-p** [comment]
I'm getting the following failure during the linux build (but the OSX build works fine):

```
squeakr-count: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by squeakr-count)
```

Any idea what might be causing this?  The executables are built successfully, but the test fails.

File: recipes/squeakr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - llvm  # [osx]
+    - bzip2
+    - zlib
+    - boost 
+    - openssl
+  run:
+    - bzip2
+    - zlib
+    - boost 
+    - openssl
```
Comment (bgruening) [inline recipes/squeakr/meta.yaml]: You need to add `- libgcc  # [not osx]` as runtime dependency.

File: recipes/squeakr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+requirements:
+  build:
+    - gcc  # [not osx]
+    - llvm  # [osx]
+    - bzip2
+    - zlib
+    - boost 
+    - openssl
+  run:
+    - bzip2
```
Comment (bgruening) [inline recipes/squeakr/meta.yaml]: Any cou probably want to pin zlib, boost and bzip2 as well.

File: recipes/squeakr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  name : {{ name|lower }}
+  version : {{ version }}
+
+source:
+  url : https://github.com/splatlab/squeakr/archive/v{{ version }}.tar.gz
+  fn : {{ name|lower }}_{{ version }}.tar.gz
+  md5 : {{ md5 }}
+
+build:
+  number : 1
```
Comment (bgruening) [inline recipes/squeakr/meta.yaml]: build numbers can start with 0

File: recipes/squeakr/meta.yaml (hunk 2 above)
Comment (rob-p) [reply recipes/squeakr/meta.yaml]: What's the easiest way to do that --- is there a "current version" variable I can use, or should I hard code the versions?

Comment (bgruening) [reply recipes/squeakr/meta.yaml]:
```
- zlib {{CONDA_ZLIB}}*
```
https://github.com/bioconda/bioconda-recipes/blob/b1a58891842d9f7f7945b20f5d4701a435879a51/recipes/pybedtools/0.6.9/meta.yaml

and boost:
`- boost {{CONDA_BOOST}}*`

Comment (bgruening) [reply recipes/squeakr/meta.yaml]: I will include a variable for bzip2 as well.

File: recipes/squeakr/meta.yaml (hunk 1 above)
Comment (rob-p) [reply recipes/squeakr/meta.yaml]: looks like this does the trick.  I will push the fix as soon as the `{{CONDA_BZIP2}}` variable is merged.

**Pull request merged**